source/develop/release-management/features/storage/4k-blocks.md

```diff
@@ -0,0 +1,93 @@
+---
+title: 4k block device support for the storage domains
+category: feature
+authors: dchaplyg
+feature_name: 4k block device support for the storage domains
+feature_modules: vdsm
+feature_status: Planning
```

Comment:
@duck-rh is above header ok? or is ti still something inherited from wiki conversion we should drop?

Reply:
> should drop?

if we don't drop it now, we will be soon :)

Reply:
The header. The things between the ---

Reply:
@akashihi i looked into this. Indeed you can delete these 3 lines

feature_name: 4k block device support for the storage domains
feature_modules: vdsm
feature_status: Planning

Reply:
I'm pretty sure we'll remove that features listing thing. Maybe even move 'features' (design documents) into the new ovirt-design repo. Not 100% sure yet.

so @akashihi you can leave the header.

Reply:
I was not aware of this new repo. It's your decision to make but I would suggest using the domain name instead of the GH domain. I think it is now possible to using GH pages with HTTPS and custom domains. We can also build/publish on OSAS servers the same way the website is done (with Jekyll instead of Middleman).

In this case I guess these headers can be removed but doing this while migrating the page instead of breaking the feature listing seems a better option.

Reply:
> breaking the feature listing

I wouldn't agree with the word "break." There are tons of problems with feature pages that I'm not going to go into here. I'll soon send a comprehensive email to devel about my plans to improve the site's information architecture.

Reply:
I'm not a developer, so not on this list, so could you Cc/forward this to me please?

Reply:
maybe even the infra list could be interested

Reply:
Yes, when I get around to sending it, I will include you.